### PR TITLE
Dev hope

### PR DESCRIPTION
**Pull request by hopesgit**
I don't remember what all I did, but I fixed quite a few methods that weren't passing spec and are now. Moved over quite a few tests, not sure that I got them all.

### CODE REVIEW

File: lib/game_teams_manager.rb [hunk 1]
```diff
@@ -18,6 +19,12 @@ def create_game_teams(game_teams_table)
     end
   end
 
+  def total_score(filtered_game_teams = @game_teams)
+    filtered_game_teams.count do |game_team|
+      game_team.goals
+    end
+  end
+
```
Comment (jdcarey128) [inline lib/game_teams_manager.rb:27]: This is a method that exists in games manager. I'm wondering if it's more efficient code-wise to just call on that method? Maybe just make a note of it for now.

File: lib/game_teams_manager.rb [hunk 2]
```diff
@@ -99,15 +105,10 @@ def game_teams_by_season(season)
...
-    team_season_tackles = {}
-    game_teams_by_season(season).each do |game|
-      if team_season_tackles[game.team_id]
-        team_season_tackles[game.team_id] += game.tackles
-      else
-        team_season_tackles[game.team_id] = game.tackles
-      end
+    game_teams_by_season(season).reduce(Hash.new(0)) do |team_season_tackles, game|
+      team_season_tackles[game.team_id] += game.tackles
+      team_season_tackles
```
Comment (jdcarey128) [inline lib/game_teams_manager.rb:110]: This looks much better. Thanks for cleaning it up!

File: lib/game_teams_manager.rb [hunk 3]
```diff
@@ -122,4 +123,68 @@ def fewest_tackles(season)
...
+      end
+      result
+    end
+  end
+
+  def game_ids_per_season(season)
+    @stat_tracker.seasonal_game_data[season].map do |games|
+      games.game_id
+    end
+  end
```
Comment (jdcarey128) [inline lib/game_teams_manager.rb:147]: This is already defined in stat_tracker (see #game_ids_per_season). This can be refactored to just call on that method.

File: lib/games_manager.rb [hunk 4]
```diff
@@ -32,14 +32,21 @@ def highest_total_score
     end.total_game_score
   end
 
+  def percentage_ties
+    ties = @games.count do |game|
+      !game.home_is_winner? && !game.visitor_is_winner?
```
Comment (jdcarey128) [inline lib/games_manager.rb:37]: The game #winner_id method will return 'tie' for games that are a tie.

**Comment by jakeheft** [comment]
This is a huge PR (almost 500 lines change); do we want to get into a practice of putting up PRs more often so they’re quicker to sift through?

File: lib/game_teams_manager.rb (hunk 3 above)
Comment (hopesgit) [reply lib/game_teams_manager.rb:147]: I opted to move it out of stat_tracker and call it in gt manager instead

File: lib/game_teams_manager.rb (hunk 1 above)
Comment (hopesgit) [reply lib/game_teams_manager.rb:27]: Well, I don't see it in the games manager... I see lowest_total_score and highest_total_score

File: lib/games_manager.rb (hunk 4 above)
Comment (hopesgit) [reply lib/games_manager.rb:37]: I'm inclined to agree, but winner_id already calls both other methods, so I don't see a difference from an efficiency standpoint.

File: lib/game_teams_manager.rb (hunk 2 above)
Comment (hopesgit) [reply lib/game_teams_manager.rb:110]: Happy to do so!

**Comment by jakeheft** [comment]
Great work, Hope. I didn't see anything that Joshua didn't point out, but nothing that needs changing. There will be some duplication with my branch for team_info method.

**Review by jakeheft — APPROVED** [review]

File: lib/team.rb [hunk 5]
```diff
@@ -6,15 +6,13 @@ class Team
               :franchise_id,
               :team_name,
               :abbreviation,
-              :stadium
```
Comment (jakeheft) [inline lib/team.rb:9]: Why was this deleted?

File: lib/team.rb [hunk 6]
```diff
@@ -6,15 +6,13 @@ class Team
...
               :link
 
   def initialize(data)
     @team_id = data[:team_id]
     @franchise_id = data[:franchiseid]
     @team_name = data[:teamname]
     @abbreviation = data[:abbreviation]
-    @stadium = data[:stadium]
```
Comment (jakeheft) [inline lib/team.rb:17]: Why was this deleted?

**Review by dunlapww — APPROVED** [review]
Look good Hope, i'll merge this in!
